scw/custom_types.go

```diff
@@ -238,11 +238,12 @@ func (n *IPNet) UnmarshalJSON(b []byte) error {
 		str += "/128"
 	}
 
-	_, value, err := net.ParseCIDR(str)
+	ip, value, err := net.ParseCIDR(str)
 	if err != nil {
 		return err
 	}
 	n.IPNet = *value
+	n.IP = ip
```

Comment:
does something like that work too?
```suggestion
	value.IP = ip
	n.IPNet = *value
```